cmd/crc/cmd/start.go

```diff
@@ -93,6 +95,19 @@ func runStart(ctx context.Context) (*types.StartResult, error) {
 		}
 	}
 
+	if runtime.GOOS == "windows" {
+		// config SharedDirPassword ('shared-dir-password') only exists in windows
+		startConfig.SharedDirPassword = config.Get(crcConfig.SharedDirPassword).AsString()
+		startConfig.SharedDirUsername = func() string {
+			u, err := user.Current()
+			if err != nil {
+				logging.Debugf("Error getting username needed for shared dirs: %v", err)
+				return ""
+			}
+			return strings.Split(u.Username, "\\")[1]
```

Comment:
the username is of the form `hostname\username`. i was hoping that it should be fine, because we can depend on std library that the o/p of `u.Username` will not change? but doesn't hurt to add a length check for the o/p of strings.Split